module/VuFind/src/VuFind/GeoFeatures/MapSelectionConfig.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * MapSelection Configuration Module
+ *
+ * PHP version 5
+ *
+ * Copyright (C) Villanova University 2010.
```

Comment:
Sure thing. I've updated all the files in the VuFind/src/VuFind/GeoFeatures directory. I'm happy to update the rest of the geographic feature code too. Please let me know if you would like me to do that.

Reply:
My global search-and-replace should have taken care of the PHP version change in all existing files, so you should only need to worry about this for newly-added files. Similarly, most of our file copyright dates reflect the date the file was first created, so it might be worth updating these in new files, but you don't have to worry about existing ones (and it's purely cosmetic anyway).